app/controllers/verify/authentications_controller.rb

```diff
@@ -59,12 +59,16 @@ def report_redacted_response
       Rollbar.debug("Verify::RedactedResponse", parameters: redacted_response.parameters)
     end
 
+    def current_policy_routing_name
```

Comment:
Would you mind rewording this commit? (92d91cba47917c389d6d2fc4dacc626a8ab14dad). I'm finding it a bit tricky to follow when reading the commits in order. It might even be better to combine withe the subsequent commit, as I'm finding it difficult to understand why this commit was added in isolation

Reply:
Good shout, commits combined. Does the new commit make more sense now?

Reply:
Yeah, that's great 👍